### DISCUSSION TITLE
Personal Access Tokens - where are these stored

### DISCUSSION THREAD

**Discussion by con-cis** [six7/figma-tokens#2356] [Q&A] [answered]
Hi, I have a question about the Personal Access Tokens for synchronising the Design Tokens with Gitlab or GitHub: On which site are they stored? In Figma or somewhere on the Token Studios site?

**Comment by six7** [comment, marked as answer]
Hey! Anything you store in the plugin is stored locally in Figma's clientStorage space scoped to the TS plugin to be used by you only, and only by the TS plugin. We do not store any of the credentials you add anywhere else. Tokens are stored on the sharedPluginData space on the file - or on your git providers if you're using sync with Git.
